ci/docker/build.Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN apt-get -yqq update && \
     apt-get -yqq install --no-install-recommends \
     software-properties-common \
     build-essential gfortran \
-    autoconf automake \
+    autoconf automake ninja-build pkg-config \
```

Comment:
Is this not already on master? :thinking: Could you rebase/merge with latest master?